docs/sphinx_qtile.py

```diff
@@ -52,6 +55,27 @@
 
         Supported bar orientations: {{ obj.orientations }}
     {% endif %}
+    {% if is_widget and screen_shots %}
+    .. raw:: html
```

Comment:
I really struggled with any of the helper ways to generate a table so resorted to raw html.